src/span.js

```diff
@@ -19,52 +19,49 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+import _ from 'lodash';
 import * as constants from './constants.js';
-import * as thrift from './thrift.js';
 import Utils from './util.js';
 import SpanContext from './span_context.js';
+import TracerLogger from './tracer_logger.js';
 
 export default class Span {
     _tracer: any;
     _name: string;
     _spanContext: SpanContext;
     _start: number;
+    _logger: TracerLogger;
     _duration: number;
     _firstInProcess: boolean;
     _isClient: boolean;
     _peer: Endpoint;
-    _annotations: Array<Annotation>;
-    _binaryAnnotations: Array<BinaryAnnotation>;
-    _baggageHeaderCache: any;
+    _logs: Array<LogData>;
+    _tags: any;
```

Comment:
I would really like to see some profiling results. iirc the reason we had the old structure here was to avoid copying the data into another set of objects when generating thrift message.  This new format will work better once we upgrade to a new data model (which we *could* do as part of UDP reporter).